TFNetworkRecLayer.py

```diff
@@ -5955,3 +5955,72 @@ def _get_output_native_rec_op(self, cell):
       y = y[0]
 
     return y
+
+
+class ZoneoutLSTMCell(BaseRNNCell):
+  '''Wrapper for tf LSTM to create Zoneout LSTM Cell
+
+  inspired by:
+  https://github.com/teganmaharaj/zoneout/blob/master/zoneout_tensorflow.py
+
+  Published by one of 'https://arxiv.org/pdf/1606.01305.pdf' paper writers.
+
+  Many thanks to @Ondal90 for pointing this out. You sir are a hero!
```

Comment:
This comment is from you, or from Tacotron?
Better adapt this, and add a reference to the related Tacotron code.
Also, use `"""` instead of `'''` (keep the code consistent; also, please check PyCharm warnings).